src/libhirte/common/common.c

```diff
@@ -0,0 +1,21 @@
+#include "common.h"
```

Comment:
The implemented functions here are declared in `protocol.h` which is included by `common.h`. Should this file be named `protocol.c`?

Reply:
I dunno, i just feel like there are way to many small scattered files. Not just here, but in many places, like all the files in bus/* with a single function each. That all would make a lot of sense with a single bus.h and bus.c. And similarly I think we could have a single common.c until it is large enough to be hard to read.

Reply:
But, thats just me, it could easily go in protocol.c too.

Reply:
Lemme change it for now, and we can maybe consolidate later.

Reply:
I agree. The bus files could/should be combined into one file. Being fairly new to C, I still struggle a bit in structuring the files.